changes/1307.bugfix.md

```diff
@@ -1 +1 @@
-Wrong typehint for `InviteGuild.features`.
+Fix error "charset must not be in content type" with InteractionServer
```

Comment:
You need to create a new changelog file for this under the name `1320.bugfix.md`, not edit an already existing one

Reply:
Fixed, I was confused about how this changes/ stuff was working